### PR TITLE
[zh-cn]: sync translation for `BatteryManager.levelchange`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

part of the #18191 

see also https://developer.mozilla.org/en-US/docs/Web/API/BatteryManager/levelchange_event

### CODE REVIEW

File: files/zh-cn/web/api/batterymanager/levelchange_event/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: BatteryManager：levelchange 事件
+slug: Web/API/BatteryManager/levelchange
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/levelchange_event/index.md:3]:
```suggestion
slug: Web/API/BatteryManager/levelchange_event
```

File: files/zh-cn/web/api/batterymanager/levelchange_event/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  sourceCommit: 8d54a21ae2677dba11569e7b7d918eac828af0b3
+---
+
+{{ApiRef("Battery API")}}{{securecontext_header}}
+
+{{domxref("BatteryManager")}} 接口的 **`levelchange`** 事件在电池的 {{domxref("BatteryManager.level", "level")}} 属性更新时触发。
+
+## 语法
+
+在 {{domxref("EventTarget.addEventListener", "addEventListener()")}} 等方法中使用事件名称，或设置事件处理程序属性。
```
Comment (yin1999) [inline files/zh-cn/web/api/batterymanager/levelchange_event/index.md:14]:
```suggestion
在 {{domxref("EventTarget.addEventListener", "addEventListener()")}} 等方法中使用事件名称，或设置事件处理器属性。
```

**Review by yin1999 — COMMENTED** [review]
nit

File: files/zh-cn/web/api/batterymanager/levelchange_event/index.md (hunk 2 above)
Comment (skyclouds2001) [reply files/zh-cn/web/api/batterymanager/levelchange_event/index.md:14]: 啊

**Review by yin1999 — APPROVED** [review]
Thank you @skyclouds2001

**Pull request merged**